src/lib/wc.js

```diff
@@ -0,0 +1,16 @@
+const wc = data => {
+  let lines = 0
+  let words = 0
+  let letters = 0
+
+  const rows = data.split('\n')
+  lines = rows.length
+  words = rows.reduce((acc, row) => acc + row.match(/\S+/g).length, 0)
```

Comment:
This logic is a little flawed. I guess you were looking for `[a-zA-Z0-9]+` instead of `\S+`.